models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -158,22 +154,22 @@ LEFT JOIN {{ ref('tokens_erc20') }} erc20b
     AND erc20b.blockchain = 'avalanche_c'
 LEFT JOIN {{ source('prices', 'usd') }} p_bought
     ON p_bought.minute = date_trunc('minute', kyberswap_dex.block_time)
-    AND p_bought.contract_address = kyberswap_dex.token_bought_address
+    AND cast(p_bought.contract_address as varbinary) = kyberswap_dex.token_bought_address
```

Comment:
is this cast necessary? the field should be varbinary already, if not, that is an issue